packages/server/api/src/app/flows/flow/flow.service.ts

```diff
@@ -67,6 +68,27 @@ export const flowService = {
     connectionIds,
     isSample,
   }: CreateFromTemplateParams): Promise<PopulatedFlow> {
```

Comment:
undo this, it's not needed in this func

Reply:
You didn't do the changes.

Remove calling validateWorkflowImport from createFromTemplate, and instead call it in create and update methods. Reuse as much as possible and as minimal changes as possible,  if create and update call a common func, then add it there.

Reply:
You didn't change the code, I don't see any usages of validateWorkflowImport